### ISSUE TITLE
Implement and test hole filling for devices 

### ISSUE THREAD

**Issue by JeffersonAero** [RMDLO/COCOpen-OpenCV#43]
Device images show holes in the ethernet ports since these ports are shadowed and appear black. We should fill these holes as part of the auto-labeling process. 

![image](https://user-images.githubusercontent.com/40707210/207457568-fabb1bb2-8689-4a86-a06a-46af474baa5b.png)

This link details how to do this:
https://learnopencv.com/filling-holes-in-an-image-using-opencv-python-c/

**Comment by ambulocetus18** [comment]
I implemented this in commit [9396a7a](https://github.com/RMDLO/COCOpen-OpenCV/commit/9396a7afe734826009ef76ab305d3b66d6d48c5b), but I haven't tested it with the dataset validation notebook.

**Comment by JeffersonAero** [comment]
Yup, this is seems to be working (See example images below). Thanks!

Working example:
![3](https://user-images.githubusercontent.com/40707210/207628555-98ad88ab-6321-47d6-bbc3-26028fcebe31.png)
![3](https://user-images.githubusercontent.com/40707210/207628571-1316f22a-ce76-4e2e-9f8b-9033d10625dd.png)

However I am now seeing holes in other images - any ideas on why this could be happening?
![9](https://user-images.githubusercontent.com/40707210/207628868-adda76b3-5be8-44ec-9174-f34cf1930efa.png)
![9](https://user-images.githubusercontent.com/40707210/207629045-c61f7d5d-1f97-4308-bb44-caa4f2836cfb.png)
